### PR TITLE
chore: update docs for alert + migration

### PR DESCRIPTION
**Pull request by nmerget**

### CODE REVIEW

File: packages/components/src/components/alert/docs/Migration.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+> **Note**
+> For a general installation or migration process check out this [documentation](https://www.npmjs.com/package/@db-ui/components).
+
+## DB UI Core ➡ DB UI Components
+
+### class
+
+| Before             | Status | After      | Description |
+| ------------------ | :----: | ---------- | ----------- |
+| `elm-notification` |   🔁   | `db-alert` |             |
```
Comment (mfranzke) [inline packages/components/src/components/alert/docs/Migration.md:12]: I'm struggling with this as notification is actually still within the backlog of components.

Comment (nmerget) [reply packages/components/src/components/alert/docs/Migration.md:12]: Well notification in v2 was both `alert` and `notification` in v3. Maybe we can combine them in v3 as well?

Comment (mfranzke) [reply packages/components/src/components/alert/docs/Migration.md:12]: than let's mention this as a comment.

Comment (nmerget) [reply packages/components/src/components/alert/docs/Migration.md]: Added a description with links to mapo

File: packages/components/src/components/alert/docs/Migration.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+| ------------------ | :----: | ---------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `elm-notification` |   🔁   | `db-alert` | The old notification component has been split into 2 different components check [Notification](https://marketingportal.extranet.deutschebahn.com/marketingportal/Design-Anwendungen/db-ux-design-system-v3/components/feedback/notification) and [Alert](https://marketingportal.extranet.deutschebahn.com/marketingportal/Design-Anwendungen/db-ux-design-system-v3/components/feedback/alert) for more information (when and how to use it). |
```
Comment (mfranzke) [inline packages/components/src/components/alert/docs/Migration.md:12]:
```suggestion
| `elm-notification` |   🔁   | `db-alert` | The old notification component has been split into 2 different components [Notification](https://marketingportal.extranet.deutschebahn.com/marketingportal/Design-Anwendungen/db-ux-design-system-v3/components/feedback/notification) and [Alert](https://marketingportal.extranet.deutschebahn.com/marketingportal/Design-Anwendungen/db-ux-design-system-v3/components/feedback/alert) for more information (when and how to use it). |
```

Probably the word _check_ was only a copy & paste in here.

**Review by mfranzke — APPROVED** [review]

**Review by mfranzke — DISMISSED** [review]

**Review by mfranzke — APPROVED** [review]

**Pull request merged**